utils/liquibase.js

```diff
@@ -129,12 +129,8 @@ function parseLiquibaseColumnType(entity, field) {
   }
 
   if (fieldType === BYTES && field.fieldTypeBlobContent !== TEXT) {
-    const { prodDatabaseType } = entity;
-    if (prodDatabaseType === MYSQL || prodDatabaseType === POSTGRESQL || prodDatabaseType === MARIADB) {
-      return 'longblob';
-    }
-
-    return 'blob';
+    // eslint-disable-next-line no-template-curly-in-string
+    return '${blobType}';
```

Comment:
Not so sure what I missed there, but it seems that builds with Blobtypes are generating less data.